src/coreclr/jit/instr.h

```diff
@@ -371,7 +371,7 @@ enum insTupleType : uint16_t
     INS_TT_NONE             = 0x0000,
     INS_TT_FULL             = 0x0001,
     INS_TT_HALF             = 0x0002,
-    INS_TT_IS_BROADCAST     = 0x0003,
+    INS_TT_IS_BROADCAST     = static_cast<uint16_t>(INS_TT_FULL | INS_TT_HALF),
```

Comment:
can you explain me why uint16 cast is needed for this specific one? I though it's just `0x1 | 0x2`

Reply:
ah, nvm, I see